include/mbgl/storage/resource.hpp

```diff
@@ -11,15 +11,7 @@ namespace mbgl {
 
 class Resource {
 public:
-    enum Kind : uint8_t {
```

Comment:
I could not find a clang-format option that does this, but we could assign values (we probably should do that anyway), which will keep them on a separate line.